governance.md

```diff
@@ -225,26 +223,27 @@ most of the time, and explicit BDFL intervention is sought only if the
 committee disagrees with that delegate’s decision and no resolution is possible
 within the team. This is different from a BDFL delegate for a specific decision
 (or a recusal situation), where the BDFL is literally giving up his/her
-authority to someone else in full. It’s more like what Linus Torvalds uses with his
-“lieutenants” model.
+authority to someone else in full.
 
 ### NumFOCUS Subcommittee
 
 The Council will maintain one narrowly focused subcommittee to manage its
 interactions with NumFOCUS.
 
--   The NumFOCUS Subcommittee is comprised of 5 persons who manage project
+-   The NumFOCUS Subcommittee is comprised of 4 persons who manage project
     funding that comes through NumFOCUS. It is expected that these funds will
     be spent in a manner that is consistent with the non-profit mission of
     NumFOCUS and the direction of the Project as determined by the full
     Council.
 -   This Subcommittee shall NOT make decisions about the direction, scope or
     technical direction of the Project.
--   This Subcommittee will have 5 members, 4 of whom will be current Council
-    Members and 1 of whom will be external to the Steering Council. No more
-    than 2 Subcommitee Members can report to one person through employment or
-    contracting work (including the reportee, i.e. the reportee + 1 is the
-    max). This avoids effective majorities resting on one person.
+-   This Subcommittee will have 4 members, at least 2 of which are
```

Comment:
I agree we should expand this to 5, but 4 is what matches the current state of the world.  I suggest that we expand it to 5 at some point later using the power this documents gives us to mutate this document.